commit 13387ac3026411b073c54f7bcf60e2335470955a (from 8099ee51a5b7d004f0dd70d78807acae9546f318)
Merge: 8099ee5 d32c405
Author: Scott Lahteine <thinkyhead@users.noreply.github.com>
Date:   Sat Jul 23 00:43:01 2016 -0700

    Merge pull request #4380 from esenapaj/Fix-the-PR-#4379
    
    Fix for the PR #4379 (Additional logging of do_blocking_move_to)

diff --git a/Marlin/Marlin_main.cpp b/Marlin/Marlin_main.cpp
index 0b6531e..72613a1 100644
--- a/Marlin/Marlin_main.cpp
+++ b/Marlin/Marlin_main.cpp
@@ -1677,7 +1677,7 @@ void do_blocking_move_to(float x, float y, float z, float fr_mm_m /*=0.0*/) {
     set_destination_to_current();          // sync destination at the start
 
     #if ENABLED(DEBUG_LEVELING_FEATURE)
-      if (DEBUGGING(LEVELING)) DEBUG_POS(PSTR("set_destination_to_current"), destination);
+      if (DEBUGGING(LEVELING)) DEBUG_POS("set_destination_to_current", destination);
     #endif
 
     // when in the danger zone
@@ -1688,7 +1688,7 @@ void do_blocking_move_to(float x, float y, float z, float fr_mm_m /*=0.0*/) {
         destination[Z_AXIS] = z;
         prepare_move_to_destination_raw(); // set_current_to_destination
         #if ENABLED(DEBUG_LEVELING_FEATURE)
-          if (DEBUGGING(LEVELING)) DEBUG_POS(PSTR("danger zone move"), current_position);
+          if (DEBUGGING(LEVELING)) DEBUG_POS("danger zone move", current_position);
         #endif
         return;
       }
@@ -1696,7 +1696,7 @@ void do_blocking_move_to(float x, float y, float z, float fr_mm_m /*=0.0*/) {
         destination[Z_AXIS] = delta_clip_start_height;
         prepare_move_to_destination_raw(); // set_current_to_destination
         #if ENABLED(DEBUG_LEVELING_FEATURE)
-          if (DEBUGGING(LEVELING)) DEBUG_POS(PSTR("zone border move"), current_position);
+          if (DEBUGGING(LEVELING)) DEBUG_POS("zone border move", current_position);
         #endif
       }
     }
@@ -1705,7 +1705,7 @@ void do_blocking_move_to(float x, float y, float z, float fr_mm_m /*=0.0*/) {
       destination[Z_AXIS] = z;
       prepare_move_to_destination_raw();   // set_current_to_destination
       #if ENABLED(DEBUG_LEVELING_FEATURE)
-        if (DEBUGGING(LEVELING)) DEBUG_POS(PSTR("z raise move"), current_position);
+        if (DEBUGGING(LEVELING)) DEBUG_POS("z raise move", current_position);
       #endif
     }
 
@@ -1713,14 +1713,14 @@ void do_blocking_move_to(float x, float y, float z, float fr_mm_m /*=0.0*/) {
     destination[Y_AXIS] = y;
     prepare_move_to_destination();         // set_current_to_destination
     #if ENABLED(DEBUG_LEVELING_FEATURE)
-      if (DEBUGGING(LEVELING)) DEBUG_POS(PSTR("xy move"), current_position);
+      if (DEBUGGING(LEVELING)) DEBUG_POS("xy move", current_position);
     #endif
 
     if (z < current_position[Z_AXIS]) {    // lowering?
       destination[Z_AXIS] = z;
       prepare_move_to_destination_raw();   // set_current_to_destination
       #if ENABLED(DEBUG_LEVELING_FEATURE)
-        if (DEBUGGING(LEVELING)) DEBUG_POS(PSTR("z lower move"), current_position);
+        if (DEBUGGING(LEVELING)) DEBUG_POS("z lower move", current_position);
       #endif
     }
 

